### ISSUE TITLE
Assessments 메뉴에서 요청사항

### ISSUE THREAD

**Issue by pak-park** [dasomx/mind-center#103]
- 19세 이상의 경우 Adult (PCL-5)의 테스트를 진행하고 다음과 같이 리스트에 결과를 표시한다.
  i.	Count_B의 값은 문항의 1-5번까지의 답의 합계이다.  
  ii.	Count_C의 값은 문항의 6-7번까지의 답의 합계이다.
  iii.	Count_D의 값은 문항의 8-14번까지의 답의 합계이다.
  iv.	Count_E의 값은 문항의 15-20번까지의 답의 합계이다.
![image](https://github.com/user-attachments/assets/3475fcbb-6b48-4574-8326-dc68a8f887be)

- 19세 미만의 경우 Children-CRTES-R의 테스트를 진행하고 다음과 같이 리스트에 결과를 표시한다.

  i.	Intrusion의 값은 문항의 1, 2, 4, 6, 10, 11, 14, 18, 19의 답의 합계이다.
  ii.	Avoidance의 값은 문항의 3, 7, 8, 9, 13, 15의 답의 합계이다.
  iii.	Arousal의 값은 문항의 5, 12, 16, 17, 20, 21, 22, 23의 답의 합계이다.
![image](https://github.com/user-attachments/assets/031e60e7-3777-4899-b899-582055159cab)

따라서 (구현예시)
- 18세인 내담자가 19세가 되었다면 19세가 되는때부터 Adult (PCL-5) 설문을 하여야 한다. 
- 따라서 Assessments는 Children-CRTES-R와 Adult (PCL-5)의 결과를 동시에 보여줄 수 있어야한다. 
- (해당 설문을 작성한것이 있다면 보여지고 없다면 안보여지도록 구현필요)

![image](https://github.com/user-attachments/assets/415038f4-0eac-4e26-a794-98b4820dd3a0)

**Comment by wisesam** [comment]
Total 말고도 세부 항목을 합계해서 리스트상에서 보여주는 것은 구현이 복잡하여 Total 만 표시되게 해야 할듯함. Edit을 눌렀을 땐 계산식으로 하는 것은 구현 할수 있음.  
-작업양: 6h(1day) 작업 필요

**Comment by wisesam** [comment]
프로파일의 Assessment Tab에 Adult, Children 별도로 리스팅하는 것은 구현이 복잡함. 한 리스트에 다 표시하고 "Form" 으로 구분하게 하는 것이 좋을 듯함

**Comment by pak-park** [comment]
> ssessment Tab에 Adult, Children 별도로 리스팅하는 것은 구현이 복잡함. 한 리스트에

데이터가 안꼬이도록 해주신다면 말씀해주시는 방법으로 가능하겠습니다.
18세 였다가 19세가 되는 경우에 대해서 고민해주십시오.

**Comment by wisesam** [comment]
네 18세를 분기로 폼을 구분하는 그러한 비즈니스 로직이 소스코드에 들어가는 것은 최소화하는 것이 좋을 것 같은데요. 원래 제가 구현했던 것 처럼 2가지 폼을 제시후 상담자가 맞는 폼을 선택하게 하는게 좋긴 합니다. 원래 상담가가 여러 폼들 중에 맞는 폼을 선택하듯이요. 추후 다른 폼들이 추가될수도 있고요. 다른 지역 Mind-Center에도 적용하기에는 그게 좋지 않나 합니다.

**Comment by pak-park** [comment]
네. 자동으로 하는것도 좋고 선택하도록 하는것도 괜찮습니다.
일단 현장 책임자에게 보고는 자동으로 하는것으로 했는데 개발팀에서 빨리 구현 가능한 방법으로 처리해주시면 되겠습니다.
그리고 이 두가지 폼은 우리 파키스탄에서는 당분간 변함없이 이용할 것이라고 합니다.
다른 사이트를 위한 개발은... 이번건 잘 완료되고 고도화하는것으로 정하면 어떨까요?
